### DISCUSSION TITLE
PoE Overlay II

### DISCUSSION THREAD

**Discussion by jylep** [Kvan7/Exiled-Exchange-2#123] [Ideas]
### Description
@Kvan7  ***With PoE Overlay II out the effort of converting Awesome Poe Trade is lessened***
***Could be a source of inspiration as well but they are a quite bigger team :)***

**Comment by Kvan7** [comment]
Sorry I'm not quite sure what you are asking or suggesting? The grammar of the sentence doesn't quite make sense. Could you also provide links to PoE Overlay II and Awesome Poe Trade? There seem to be many github repos and I don't know quite which ones you are referring to

**Comment by jylep** [comment]
Sorry I meant Awekened Poe Trade, phone converted it.

https://www.overwolf.com/app/kyusung4698-poe_overlay_ii

Like I say the effort is lessend as less people will be looking for a custom made (aka NOT overwolf based) solution but there will still be some people looking for an alternate solution. I was simply pointing this out because github does not let me message you directly

https://www.reddit.com/r/PathOfExile2/comments/1hdb1bz/poe_overlay_2_is_now_released_finally_a_price/

**Reply by Kvan7** [reply]
All good! Thanks for the links! I'll look into it but can't guarantee stuff right now since I'm still fixing bugs :)

**Reply by jylep** [reply]
Not sure you understand what I am saying.

In simple terms:

- There is a solution out there already
- Don't stress yourself and take your time
- Check the existing solution for inspiration
